lib/rubocop/cop/lint/symbol_conversion.rb

```diff
@@ -21,10 +27,26 @@ module Lint
       #   :underscored_symbol
       #   :'hyphenated-string'
       #
+      # @example EnforcedStyle: consistent
```

Comment:
Might be a good idea to add the `strict` case for hashes as well, so it's clearer what this means for them.

Reply:
@bbatsov I added additional documentation, thanks!